corehq/apps/data_dictionary/util.py

```diff
@@ -173,3 +173,13 @@ def get_data_dict_props_by_case_type(domain):
 def get_data_dict_case_types(domain):
     case_types = CaseType.objects.filter(domain=domain).values_list('name', flat=True)
     return set(case_types)
+
+
+def validated_fields(domain, case_type_name):
```

Comment:
This name sounds like it returns fields that have been validated. How about `fields_to_validate`?